libres/lib/include/ert/analysis/ies/ies_config.hpp

```diff
@@ -42,6 +42,9 @@ void set_truncation(config_type *config, double truncation);
 
 void set_option_flags(config_type *config, long flags);
 long get_option_flags(const config_type *config);
+bool get_option(const config_type *config, long option);
```

Comment:
How do I know which options I have available?
Could the parameter `option` be an enum instead of a `long`?

Reply:
> How do I know which options I have available?

The available values are here: https://github.com/equinor/ert/blob/main/libres/lib/include/ert/analysis/analysis_module.hpp

>Could the parameter option be an enum instead of a long?

Yes